ecommerce/views_test.py

```diff
@@ -693,6 +718,22 @@ def test_patch_basket_update_active_inactive_product(basket_client, basket_and_c
     assert resp.status_code == status.HTTP_200_OK
 
 
+def test_patch_basket_update_active_inactive_product_text_id(
```

Comment:
```suggestion
def test_patch_basket_update_inactive_product(
```

I'm not seeing any other test cases with this name

Reply:
Oh sorry, I mismatched `test_patch_basket_update_inactive_product` with `test_patch_basket_update_inactive_product `.

Reply:
If we are going to discard inactive from the code then we don't need for that test anymore.

Reply:
@umarmughal824 I still think it's useful to test that an inactive product raises a `ValidationError`. The text of the exception will be different now, but the exception class will be the same

Reply:
If we are going to use the `models.objects` we will never get that inactive status so why still do we need to test that inactive?

Reply:
Yes, that's the current implementation. This test case would fail if someone went in and changed that query to fetch inactive products, and it's also making it explicit that the basket will only accept active products.

Reply:
I have updated it.